README.md

```diff
@@ -66,35 +66,34 @@ With the goal of being reusable, this application is developed with a component
+All components must extend from [\yii\base\Component](http://www.yiiframework.com/doc-2.0/guide-structure-application-components.html) so that they can be configured in the ```components``` section of the application configuration. This also allows them to be accessed via ```\Yii::$app->componentId```. While each component has a defind interface for methods to implement, what properties it needs for configuration are up to each implementation as appropriate. See [our common/config/local.php.dist](https://github.com/silinternational/idp-pw-api/blob/develop/application/common/config/local.php.dist) for examples of configurations.
 
 ### Authentication Component
-We use SAML for authentication but this component can be replaced to support whatever method is needed. For example an auth component could be written to implement OAuth or use Google, etc. 
+We use SAML for authentication but this component can be replaced to support whatever method is needed. For example an auth component could be written to implement OAuth or use Google, etc.
 
 * Component ID: ```auth```
 * Implement interface: ```Sil\IdpPw\Common\Auth\AuthnInterface```
 * Example implementation: [idp-pw-api-auth-saml](https://github.com/silinternational/idp-pw-api-auth-saml)
 
 ### Password Store Component
-You can store your passwords wherever you like, whether it is LDAP, Active Directory, a database, or even Redis. 
+You can store your passwords wherever you like, whether it is LDAP, Active Directory, a database, or even Redis.
 
 * Component ID: ```passwordstore```
 * Implement interface: ```Sil\IdpPw\Common\PasswordStore\PasswordStoreInterface```
 * Example implementation: [idp-pw-api-passwordstore-ldap](https://github.com/silinternational/idp-pw-api-passwordstore-ldap)
 
 ### Personnel Component
-The personnel component is used to look up informaton about users from your companies personnel system. This includes verifying that they are an active employee, getting information about them like name, email, employee id, if they have a supervisor and what their supervisors email address is and if the personnel system is aware of spouses it can also provide the spouse's email address.
+The personnel component is used to look up informaton about users from your company's personnel system. This includes verifying that they are an active employee, getting information about them like name, email, employee id, whether they have a supervisor and what their supervisors email address is. If the personnel system is aware of spouses it can also provide the spouse's email address.
 
 * Component ID: ```personnel```
-* Implement interface: ```Sil\IdpPw\Common\Personnel\PersonnelInterface```
+* Implement interface: ```common\components\Personnel\PersonnelInterface```
```

Comment:
Since namespaces are global, we should probably have the new namespace begin with something like `\Sil\IdpPw\common\components\Personnel\...` (rather than merely using `\common\components\Personnel\..`).

Reply:
As discussed, since this is not a repo that could be easily integrated with another project, I'll leave this as is.